README.md

```diff
@@ -1 +1,246 @@
-# brainypedia-design-document
\ No newline at end of file
+# BrainKB Design Document
+<img src="logo.png" width="100px" style="float:right; margin-top:-45px"/>
+
+**Author:** Tek Raj Chhetri | <tekraj@mit.edu>
+
+## Overview
+_BrainKB serves as a knowledge base platform that provides scientists worldwide with tools for searching, exploring, and visualizing Neuroscience knowledge represented by knowledge graphs (KGs)_. Moreover, BrainKB provides cutting-edge tools that enable scientists to contribute new information (or knowledge) to the platform, ensuring it remains the go-to destination for all neuroscience-related research needs.
+
+The main objective of BrainKB is to represent Neuroscience knowledge as a knowledge graph such that it can be used for different downstream tasks, such as making predictions and new inferences in addition to querying and viewing information.  The expected outcome of the BrainKB includes the following:
```

Comment:
is there a reason why Neuroscience is always capitalized?

Reply:
no, fixed the issue